src/transitions/initiate_standby.py

```diff
@@ -1,27 +1,27 @@
 import logging
 import datetime
-from clients.kube import scale
-from clients.patroni import set_readonly_cluster, set_primary_cluster
-from clients.kube import scale, scale_and_wait, delete_pvc, delete_configmap
+from clients.kube import scale_and_wait, delete_pvc, delete_configmap
 from clients.tekton import trigger_tekton_build
 from transitions.wait_for import WaitFor
-from transitions.shared import maintenance_on, set_in_recovery, update_patroni_spilo_env_vars
+from transitions.shared import maintenance_on, scale_health_api, set_in_recovery, update_patroni_spilo_env_vars
 from config import config
 
 logger = logging.getLogger(__name__)
 
 
 def initiate_active_standby(logic_context, py_env: str):
-    scale(config.get('kube_health_namespace'), 'deployment',
-          config.get('deployment_health_api'), 0, py_env)
+    scale_health_api(config.get('kube_health_namespace'), 
+                     config.get('deployment_health_api'), 
+                     0, py_env)
     return initiate_standby(logic_context,
                             py_env, 'gold-standby')
 
 
 def initiate_passive_standby(logic_context, py_env: str):
     set_in_recovery(False, py_env)
-    scale(config.get('kube_health_namespace'), 'deployment',
-          config.get('deployment_health_api'), 2, py_env)
+    scale_health_api(config.get('kube_health_namespace'), 
+                     config.get('deployment_health_api'), 
+                     s, py_env)
```

Comment:
I think there is an `s` that should be a 2

Reply:
good catch. use suggestions!
```suggestion
                     2, py_env)
```